examples/bid-optimizer/src/tests/index.ts

```diff
@@ -1,73 +1,72 @@
-import { expect } from "chai";
-import "mocha";
-import { core } from "@mediarithmics/plugins-nodejs-sdk";
-import * as request from "supertest";
-import * as sinon from "sinon";
-import * as rp from "request-promise-native";
-import { MyBidOptimizerPlugin } from "../MyPluginImpl";
+import {expect} from 'chai';
+import 'mocha';
+import {core} from '@mediarithmics/plugins-nodejs-sdk';
+import * as request from 'supertest';
+import * as sinon from 'sinon';
+import {MyBidOptimizerPlugin} from '../MyPluginImpl';
 
-describe("Test Example BidOptimizer", function() {
-    // We stub the Gateway calls
-    const rpMockup: sinon.SinonStub = sinon.stub();
-  
-    // Activity Analyzer stub
-    const bidOptimizer: core.DataResponse<core.BidOptimizer> = {
-      status: "ok",
-      data: {
-        id: "1000",
-        name: "my analyzer",
-        organisation_id: "1000",
-        engine_version_id: "123456",
-        engine_group_id: "com.mediarithmics.visit-analyzer",
-        engine_artifact_id: "default"
-      }
-    };
-  
-    rpMockup
-      .withArgs(
-        sinon.match.has(
-          "uri",
-          sinon.match(function(value: string) {
-            return value.match(/\/v1\/bid_optimizers\/(.){1,10}/) !== null;
-          })
-        )
+describe('Test Example BidOptimizer', function () {
+  // We stub the Gateway calls
+  const rpMockup: sinon.SinonStub = sinon.stub();
+
+  // Activity Analyzer stub
+  const bidOptimizer: core.DataResponse<core.BidOptimizer> = {
+    status: 'ok',
+    data: {
+      id: '1000',
+      name: 'my analyzer',
+      organisation_id: '1000',
+      engine_version_id: '123456',
+      engine_group_id: 'com.mediarithmics.visit-analyzer',
+      engine_artifact_id: 'default'
+    }
+  };
+
+  rpMockup
+    .withArgs(
+      sinon.match.has(
+        'uri',
+        sinon.match(function (value: string) {
+          return value.match(/\/v1\/bid_optimizers\/(.){1,10}/) !== null;
+        })
       )
-      .returns(bidOptimizer);
-  
-    // Activity Analyzer properties stub
-    const bidOptimizerProperties: core.PluginPropertyResponse = {
-      count: 1,
-      data: [
-        {
-          technical_name: "name",
-          value: {
-            value:
-              "my bid optimizer",
-          },
-          property_type: "STRING",
-          origin: "PLUGIN",
-          writable: true,
-          deletable: true
-        }
-      ],
-      status: "ok"
-    };
-  
-    rpMockup
-      .withArgs(
-        sinon.match.has(
-          "uri",
-          sinon.match(function(value: string) {
-            return (
-              value.match(/\/v1\/bid_optimizers\/(.){1,10}\/properties/) !==
-              null
-            );
-          })
-        )
+    )
+    .returns(bidOptimizer);
+
+  // Activity Analyzer properties stub
```

Comment:
-> Bid Optimizer (old comment issue...)

Reply:
I prefere to keep this PR as a pure code formatting. A lot of theses comments have been removed in the next PR.

Reply:
:ok_hand: